### PR TITLE
Basic LaTEX Template

### PR DESCRIPTION
**Pull request by ashfordneil**
Not sure if we still want to use LaTEX, but if we do, this is a starting template.

### CODE REVIEW

**Comment by wisebaldone** [comment]
@matthew-lake, please review this for us boss

**Comment by wisebaldone** [comment]
Few recommendations before I can `LGTM` this.

1) Seems your are using the old license instead of the MIT license.
2) Seems your listing the University of Queensland as the authors instead of the University of Queensland Computer Society as the collective author.

other than that, looks good for a starter.

File: tex/courses/CSSE1001.tex [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+\textbf{Pre-requisite Courses} &
+
+%%% COURSE PREREQUISITES
+None
+%%%
+
+\\
+\textbf{Companion Courses} &
+
+%%% COMPANIOUN COURSES
```
Comment (TRManderson) [inline tex/courses/CSSE1001.tex]: Spelling

File: tex/misc/titlePage.tex [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+\textcolor{Red}{ % Red font color
+{\Huge UQ Computing Society}\\[0.5\baselineskip] % Title line 1
+{\Large 2017}\\[0.75\baselineskip] % Title line 2
+{\Huge Subject Guide}} % Title line 3
+
+\vspace{0.25\drop} % Whitespace between the title and short horizontal line
+\rule{0.3\textwidth}{0.4pt}\par % Short horizontal line under the title
+\vspace{\drop} % Whitespace between the thin horizontal line and the author name
+
+{\Large \textsc{The University of Queensland}}\par % Author name
```
Comment (TRManderson) [inline tex/misc/titlePage.tex]: As @wisebaldone said, pls fix

**Review by TRManderson — CHANGES_REQUESTED** [review]

**Review by TRManderson — APPROVED** [review]

**Pull request merged**